src/config/plugins.ts

```diff
@@ -0,0 +1,273 @@
+import {
+  AdjustmentsHorizontalIcon,
+  ArchiveBoxArrowDownIcon,
+  ArrowPathIcon,
+  Bars3Icon,
+  BoltIcon,
+  CameraIcon,
+  ChatBubbleLeftIcon,
+  CommandLineIcon,
+  CurrencyDollarIcon,
+  DevicePhoneMobileIcon,
+  DocumentMagnifyingGlassIcon,
+  EllipsisHorizontalIcon,
+  FingerPrintIcon,
+  GlobeAltIcon,
+  HomeIcon,
+  MapPinIcon,
+  PaintBrushIcon,
+  PhoneArrowUpRightIcon,
+  PhoneIcon,
+  PlayCircleIcon,
+  PlayIcon,
+  SpeakerWaveIcon,
+  SpeakerXMarkIcon,
+  StopIcon,
+  VideoCameraIcon,
+  WalletIcon,
+} from '@heroicons/vue/20/solid'
+
+export interface Action {
+  icon?: any
+  href: string
+  title: string
+  name?: string
+  author: string
+  description: string
+  iconForeground?: string
+  iconBackground?: string
+}
+
+export interface Plugin extends Action {
+  npmDownloads?: number
+  githubStars?: number
+  readme?: string
+}
+
+export const actions = [
+  {
+    name: '@capgo/cli',
+    author: 'github.com/riderx',
+    description: 'A CLI to upload to capgo servers',
+    href: 'https://github.com/Cap-go/capgo-cli',
```

Comment:
this list has to be dynamic from npm not static that why i asked for exclude list

Reply:
But then how we plan to inject the icons aptly for the each package?

Reply:
I can write a script which we can use pre-building to generate that list, but the issue is rate limiting + which icon to choose dynamically.

Reply:
make sense let it like that then